torch_brain/data/collate.py

```diff
@@ -110,6 +109,57 @@ def pad8_collate_object_fn(
     return _collate([e.obj for e in batch], collate_fn_map=pad8_collate_fn_map)
 
 
+# pad2d
+Padded2dObject = namedtuple("Padded2dObject", ["obj"])
+
+
+def pad2d(obj):
+    """
+    Args:
+        obj: Can be tensors, numpy arrays, lists, tuples, or dictionaries.
+    """
+    return Padded2dObject(obj)
+
+
+def track_mask2d(input: Union[torch.Tensor, np.ndarray]):
+    r"""Wrap an array or tensor to specify that its padding mask should be tracked. This
+    is used in conjunction with :obj:`pad2d`.
+
+    Args:
+        input: An array or tensor.
+    """
+    assert input.ndim == 2
```

Comment:
Would be better to raise error `if input.ndim != 2` instead of having an assertion here.